tests/unit/checkboxradio/checkboxradio_methods.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * checkboxradio_methods.js
+ */
+
+
+(function($) {
+
+module( "Checkboxradio: methods" );
+	test( "Checkbox: refresh", function() {
+		var checkbox = $( "#checkbox-method-refresh" ),
+			widget;
+		expect( 2 );
+		checkbox.checkboxradio({
+			icon: true
+		});
+
+		widget = checkbox.checkboxradio( "widget" );
+		delete $(".ui-icon" )[0];
+
+		checkbox.checkboxradio( "refresh" );
+		strictEqual( widget.find( ".ui-icon" ).length, 1, "Icon is recreated on refresh" );
+		checkbox.prop( "checked", true );
```

Comment:
Let's add a line break before this line. From here its essentially another test.